recipes/sasview/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "sasview" %}
+{% set version = "v5.0_beta.2_8_g390494d7c" %}  # git describe --tags --dirty --always | sed 's/-/_/g'
+{% set file_ext = "tar.gz" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  git_url: https://github.com/SasView/sasview
+  git_rev: 390494d7cc6099973fce721a89afc666ae254c11
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - numpy
+    - pyqt >=5
+    - python
+    - setuptools
+  run:
+    - bumps
+    - h5py >=2.5
+    - lxml
+    - matplotlib
+    - numpy
+    - periodictable >=1.5.0
+    - pillow
+    - pyparsing >=2.0.0
+    - pyqt >=5
+    - python
+    - qt5reactor
+    - sasmodels
+    - scipy
+    - twisted <=18.4
+    - xhtml2pdf
+
+test:
+  imports:
+    - sas.sasview
+    - sas.qtgui
+    - sas.qtgui.MainWindow
+  commands:
+    - which sasview
+
+about:
+  home: https://github.com/SasView/sasview
+  license: Custom
```

Comment:
Isn't this the BSD 3-clause licence? If so, can you change this to:

```yaml
  license: BSD 3-Clause
  license_family: BSD
  license_file: LICENSE.TXT
```

Reply:
It does not say explicitly it's "BSD 3-Clause", nor GitHub recognizes the type of the license. I'd leave it to the main developers.

Reply:
The text is the same as [the Wikipedia listing](https://en.wikipedia.org/wiki/BSD_licenses#3-clause_license_(%22BSD_License_2.0%22,_%22Revised_BSD_License%22,_%22New_BSD_License%22,_or_%22Modified_BSD_License%22)) excluding `<COPYRIGHT HOLDER>` minor formatting changes.

Reply:
Indeed! Thanks for the link :)

Reply:
Where do I place the `LICENSE.TXT` file?

Reply:
It should be picked up from the source tarball.

Reply:
Got it, thanks.